release_notes/versioning-policy.adoc

```diff
@@ -0,0 +1,51 @@
+[id='ocp-4-1-versioning-policy']
+= {product-title} Versioning Policy
+include::modules/common-attributes.adoc[]
+:context: release-notes
+
+toc::[]
+
+{product-title} provides strict backwards compatibility guarantees for all
+supported APIs, excluding alpha APIs (which may be changed without notice) and
+beta APIs (which may occasionally be changed in a non-backwards compatible
+manner).
+
+{product-title} 3.11 is compatible with {product-title} {product-version}. Red
+Hat did not publicly release {product-title} 4.0 and, instead, is releasing
+{product-title} {product-version} directly after version 3.11.
+
+The {product-title} version must match between master and node hosts, excluding
+temporary mismatches during cluster upgrades. For example, in a 3.11 cluster, all
+masters must be 3.11 and all nodes must be 3.11. However, {product-title} will
```

Comment:
So, in 4.1, you must get a new `oc`. I've had a couple of bugs about it, so I'm adding this note elsewhere:

```
[IMPORTANT]
====
If you installed an earlier version of `oc`, you cannot use it to complete all
of the commands in {product-title} {product-version}. You must download and
install the new version of `oc`.
====
```
Will you double-check this last sentence?